inject.js

```diff
@@ -1,65 +1,71 @@
 const nest = require('depnest');
-const index = require('./index');
-const {
-  h
-} = require('mutant');
+const { h, onceTrue } = require('mutant');
+// const { isMsg } = require('ssb-ref')
 
-var onceTrue = require('mutant/once-true');
+const index = require('./index');
 
 exports.gives = nest({
-  'app.html': {
-    page: true,
-    menuItem: true
-  }
+  'app.html.menuItem': true, 
+  'app.page.chessIndex': true,
+  'router.sync.routes': true,
 })
 
 exports.needs = nest({
+  'app.page.chessIndex': 'first',
   'sbot.obs.connection': 'first',
   'backlinks.obs.for': 'first'
 });
 
 exports.create = function(api) {
-  const route = '/chess'
-
-  const topLevelDomElement = document.createElement('div');
-  topLevelDomElement.className = "ssb-chess-container";
-
+  const topLevelDomElement = h('div.ssb-chess-container', { title: '/chess' })
   var pageLoaded = false;
 
   return nest({
-    'app.html': {
-      menuItem: menuItem,
-      page: chessPage
-    }
+    'app.html.menuItem': menuItem, 
+    'app.page.chessIndex': chessIndex,
```

Comment:
If you only ever want one patchbay tab per page, then this `app.page.chessIndex` is redundent. You can choose to note _give_ it.

I put this in to make it easier to extend the pattern should you want to